### DISCUSSION TITLE
Validation against json-schema.org/draft-07/schema#

### DISCUSSION THREAD

**Discussion by piltdown64** [josdejong/svelte-jsoneditor#406] [Q&A]
Hi,
I am using the web app jsoneditoronline and trying to validate some simple JSON against JSON schema draft-07,  but I keep seeing the error message below. If I change the -07 to -06 then the error message disappears.  Is there something I've missed?

Error: no schema with key or ref "http://json-schema.org/draft-07/schema#"

For example:  Put the following JSON. in the left panel

```
{
  "userId": 1,
  "name": "Luke Skywalker"
}
```

....with the following schema in the right panel:

```
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "title": "My test schema",
  "type": "object",
  "properties": {
    "userId": {
      "type": "number"
    },
    "name": {
      "type": "string"
    }
  },
  "required": [
    "userId",
    "name"
  ]
}
```

Thanks,
John

**Comment by josdejong** [comment]
Thanks for reporting John. 

I did some testing, and `draft-07` is supported out of the box in `svelte-jsoneditor`, but the web application https://jsoneditoronline.org has some additional configuration, and `draft-07` was missing there. This has been fixed now in JSON Editor Online v6.17.1 (you may need to reload the web application).
